Sources/MapboxNavigation/FeedbackViewController.swift

```diff
@@ -231,30 +231,48 @@ public class FeedbackViewController: UIViewController, DismissDraggable, UIGestu
         
         NSLayoutConstraint.activate(constraints)
     }
-    
+
+    /**
+     Sends the current feedback event for the specific feedback item.
+
+     - Parameter item: a feedback item for which feedback will be sent.
+     */
     func send(_ item: FeedbackItem) {
-        if let uuid = self.uuid {
+        if let uuid = currentFeedbackUUID {
```

Comment:
When does this fail? Should we handle such errors explicitly?

Reply:
Agreed to leave it as it is. Possible error handling can be done in scope of separate PR.